apps/prairielearn/public/stylesheets/workspace.css

```diff
@@ -20,3 +20,25 @@
 .popover {
   max-width: 30rem;
 }
+
+@keyframes jiggle {
+  0%,
+  40%,
+  60%,
+  100% {
+    transform: scale(100%);
+  }
+
+  45%,
+  55% {
+    transform: scale(103%);
+  }
+
+  50% {
+    transform: scale(100%);
+  }
+}
+
+.jiggle {
```

Comment:
Pedantic nit: I imagine you grabbed this from my demo, where this was originally actually a jiggle, but now it's not so much that. Maybe `pulse-scale` is a better fit?

Reply:
Changed to `pulse`